app/js/lib/foxbox/db.js

```diff
@@ -13,15 +27,18 @@ const DB_TAG_STORE = 'tags';
 
 export default class Db {
   constructor() {
-    this.db = null;
+    // Private properties.
+    this[p.db] = null;
+
+    return Object.seal(this);
```

Comment:
Honestly I have a mixed feelings about sealing in the constructor, but don't have strong opinion either. Nevertheless I'm open to try :)

P.S This way we "mark" class as non-extendable (maybe that's what we want). In my head, the owner of the instance is the one who should or not seal/freeze the object, but let's see.